### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by Thomasdezeeuw**

### CODE REVIEW

File: go-fuzz/cover.go [hunk 1]
```diff
@@ -60,35 +60,18 @@ func updateMaxCover(base, cur []byte) int {
...
+		return 16
 	} else if x <= 32 {
-		x = 32
+		return 32
 	} else if x <= 64 {
-		x = 64
-	} else {
-		x = 255
-	}
-	if !*flagCoverCounters {
```
Comment (dvyukov) [inline go-fuzz/cover.go]: Why did you remove flagCoverCounters? It is useful.

File: gen/main.go [hunk 2]
```diff
@@ -33,14 +33,23 @@ func Rand(n int) int {
 }
 
 func Emit(data, hint []byte, valid bool) {
-	//fmt.Printf("INPUT [%v]%q\n", len(data), data)
-	f, err := os.Create(filepath.Join(*flagOut, fmt.Sprintf("%v", seq)))
+	filePath := filepath.Join(*flagOut, fmt.Sprintf("%d", seq))
+	f, err := os.Create(filePath)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to create file: %v\n", err)
+		fmt.Fprintf(os.Stderr, "failed to create file: %s\n", err.Error())
```
Comment (dvyukov) [inline gen/main.go]: What is wrong with the current code? Error is formatted fine with %v

File: gen/main.go [hunk 3]
```diff
@@ -33,14 +33,23 @@ func Rand(n int) int {
 }
 
 func Emit(data, hint []byte, valid bool) {
-	//fmt.Printf("INPUT [%v]%q\n", len(data), data)
-	f, err := os.Create(filepath.Join(*flagOut, fmt.Sprintf("%v", seq)))
+	filePath := filepath.Join(*flagOut, fmt.Sprintf("%d", seq))
```
Comment (dvyukov) [inline gen/main.go]: What is wrong with the current code?

File: gen/main.go [hunk 4]
```diff
@@ -33,14 +33,23 @@ func Rand(n int) int {
...
+		fmt.Fprintf(os.Stderr, "failed to create file: %s\n", err.Error())
 		os.Exit(1)
 	}
-	f.Write(data)
-	f.Close()
+	defer f.Close()
+
+	n, err := f.Write(data)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create file: %s\n", err.Error())
```
Comment (dvyukov) [inline gen/main.go]: s/create/write/
format err with %v

File: go-fuzz/hub.go [hunk 5]
```diff
@@ -311,8 +311,6 @@ func (hub *Hub) updateScores() {
 			score /= 10
 		} else if execTime > 4 {
 			score /= 4
-		} else if execTime > 4 {
```
Comment (dvyukov) [inline go-fuzz/hub.go]: Why did you remove this?

Comment (dgryski) [inline go-fuzz/hub.go]: It duplicates the if test just above it and will never fire.

Comment (dvyukov) [inline go-fuzz/hub.go]: right

File: go-fuzz/cover.go (hunk 1 above)
Comment (Thomasdezeeuw) [inline go-fuzz/cover.go]: Because at that point is always 255 anyway.

Comment (Thomasdezeeuw) [inline go-fuzz/cover.go]: Wait you're right, I'll update the pr.

Comment (Thomasdezeeuw) [inline go-fuzz/cover.go]: Fixed it.

**Comment by Thomasdezeeuw** [comment]
Updated to according to your comments.

**Comment by dvyukov** [comment]
thanks!
looks good to me now
